networking/network_observability/network-observability-operator-monitoring.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="monitoring-network-observability-operator"]
+= Monitoring the Network Observability Operator
+include::_attributes/common-attributes.adoc[]
+:context: network_observability
+
+toc::[]
+You can use the {product-title} web console to monitor metrics related to the health of the Network Observability Operator. 
```

Comment:
Nitpicking: the dashboard to consult infra health metrics will not be part of the netobserv console plugin but will be a new dashboard in the monitoring operator section. Links from the netobserv console plugin will be provided.

Reply:
Okay...not a nitpick I think I'm generally a little confused about the different dashboards and what information is where. Thanks for clarifying this point. I _think_ I'm understanding.

Reply:
@OlivierCazade I rewatched the demo recording, and I guess I'm confused still lol. What I'm doing here is creating a tab to document information about dashboards, monitoring, and alerting. For now, I guess, Alerting is the only thing I'm documenting, which is the health alert in Observe->Alerting. In the future, when Kalman's metrics work is live, then I was thinking to add that information here as well.

Reply:
@OlivierCazade Is this part accurate/complete now? Let me know if you think any details are missing.